### PR TITLE
Update project using ember-cli-update

### PR DESCRIPTION
**Pull request by amk221**
Updates the project using ember-cli-update 
Removes this warning

>  WARN: DEPRECATION: Versions of modifier manager capabilities prior to 3.22 have been deprecated. You must update to the 3.22 capabilities. [deprecation id: manager-capabilities.modifiers-3-13] See https://emberjs.com/deprecations/v3.x#toc_manager-capabilities-modifiers-3-13 for more details.

### CODE REVIEW

**Review by wongpeiyi — COMMENTED** [review]
Looks good, just a couple of questions

File: addon/templates/components/route.hbs [hunk 1]
```diff
@@ -1,22 +1,21 @@
 {{#let
-  (component this.componentName
-    model=this.renderTask.attributes
-    __renderTask__=this.renderTask
-  )
+  (component this.componentName model=this.renderTask.attributes __renderTask__=this.renderTask)
```
[the remark this answers is not in the document]
Comment (amk221) [reply addon/templates/components/route.hbs:2]: Due to the ember-cli linting dependancies being updated, there were .js and .hbs lint warnings. So I ran `prettier . --fix`.

File: README.md [hunk 2]
```diff
@@ -4,9 +4,9 @@
 
 ## Compatibility
 
-- Ember.js v3.16 or above
-- Ember CLI v2.13 or above
-- Node.js v10 or above
+- Ember.js v3.24 or above
+- Ember CLI v3.24 or above
+- Node.js v14 or above
```
[the remark this answers is not in the document]
Comment (amk221) [reply README.md]: I accepted the incoming changes from ember-cli-update. But, I ran `ember-try` just now with a 3.16 scenario, and it passed. So I will revert this change

File: config/ember-try.js [hunk 3]
```diff
@@ -5,7 +5,24 @@ const { embroiderSafe, embroiderOptimized } = require('@embroider/test-setup');
...
+        }
+      },
+      {
+        name: 'ember-lts-3.24',
+        npm: {
+          devDependencies: {
+            'ember-source': '~3.24.3'
+          }
+        }
+      },
```
Comment (wongpeiyi) [inline config/ember-try.js:25]: Looks like a mistake

**Comment by wongpeiyi** [comment]
Thanks, I've tested it and seems ok. Please squash the last two commits into the first and I'll merge.

**Comment by amk221** [comment]
Ok thanks, done

**Pull request merged**